inc/saiacl.h

```diff
@@ -2331,6 +2337,26 @@ typedef enum _sai_acl_entry_attr_t
      */
     SAI_ACL_ENTRY_ATTR_ACTION_MACSEC_FLOW,
 
+    /**
+     * @brief LAG Hash ID
+     *
+     * @type sai_acl_action_data_t sai_object_id_t
+     * @flags CREATE_AND_SET
+     * @objects SAI_OBJECT_TYPE_HASH
+     * @default disabled
+     */
+    SAI_ACL_ENTRY_ATTR_ACTION_SET_LAG_HASH_ID,
+
+    /**
+     * @brief ECMP Hash ID
+     *
+     * @type sai_acl_action_data_t sai_object_id_t
+     * @flags CREATE_AND_SET
+     * @objects SAI_OBJECT_TYPE_HASH
+     * @default disabled
+     */
+    SAI_ACL_ENTRY_ATTR_ACTION_SET_ECMP_HASH_ID,
+
```

Comment:
In addition to ACL/Policy setting LAG+ECMP hash object, we can also add Hash object to LAG and Nextgroup objects.